code/online_challenges/src/project_euler/problem_034/problem_034.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <vector>
+int factorial (int n) 
+{
+    if(n == 0)
+        return 1;
+    return n * factorial(n - 1);
+}
+
+int main()
+{
+    std::vector<int> factor(10);
+    for(int i = 0; i < 10; ++i)
+        factor[i] = factorial(i);
+    int num = 3, temp, sum = 0;
+    while (num < 2540162)
```

Comment:
Where did you get this number from? If it has a significant meaning, assign it to a `constexpr` constant.